teams/wg-prioritization.toml

```
@@ -0,0 +1,15 @@
+name = "wg-prioritization"
+subteam-of = "compiler"
+wg = true
+
+[people]
+leads = ["spastorino", "wesleywiser"]
+members = ["spastorino", "wesleywiser", "pnkfelix", "centril", "LeSeulArtichaut", "JohnTitor", "Dylan-DPC"]
```

```suggestion
members = ["spastorino", "wesleywiser", "pnkfelix", "Centril", "LeSeulArtichaut", "JohnTitor", "Dylan-DPC"]
```

Note that it's case-sensitive here.

---

Fixedm, thanks!.